.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Reason of why I added this (for now):
The `pinia-class` package has a peer dependency for `vue-facing-decorator` of version `^2.1.20`.
The version I'm currently using is `^3.0.3`. The reason is we cannot extend mixins in version `2.x` in `vue-facing-decorator`.

Please take a look at package.json to see the versions.

Reply:
Yes, we should get away from extending mixins (if we still do that).

Reply:
I 100% agree Sev. I think going away from that is a great idea since this is not the preferable way to do it using vue-facing-decorator. It's going to be a large amount of redundant work.